bash-preexec.sh

```diff
@@ -38,10 +38,14 @@ if [[ -z "${BASH_VERSION:-}" ]]; then
 fi
 
 # Avoid duplicate inclusion
-if [[ "${__bp_imported:-}" == "defined" ]]; then
+if [[ "${bp_imported:-}" == "defined" ]]; then
     return 0
 fi
-__bp_imported="defined"
+bp_imported="defined"
+
+# WARNING: This variable has never been designed as part of the public API. Bash
```

Comment:
How about just making this a deprecation warning, e.g. "This variable is no longer used and should not be relied upon, use bp_imported instead."?